webpack.config.js

```diff
@@ -31,15 +40,11 @@ module.exports = {
       },
       {
         test: /\.css$/,
-        use: [
-          { loader: "style-loader" },
-          {
-            loader: "css-loader",
-            options: {
-              modules: true,
-            },
-          },
-        ],
+        use: ["style-loader", cssLoader],
```

Comment:
I think the `modules` option is not on by default. Intentional to remove it?

Reply:
restored the module option